services/api/src/models/payment-bank-account.model.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  PaymentBankAccountSchema,
+  PaymentBankAccountStatus,
+} from '@algomart/schemas'
+import { Model } from 'objection'
+
+import { BaseModel } from './base.model'
+import { UserAccountModel } from './user-account.model'
+
+export class PaymentBankAccountModel extends BaseModel {
+  static tableName = 'PaymentBankAccount'
+  static jsonSchema = PaymentBankAccountSchema
+
+  ownerId!: string
+  externalId!: string
+  status!: PaymentBankAccountStatus | null
```

Comment:
The `status` column in the migration is marked as not nullable. So you'll wanna remove `| null` here or remove the `.notNullable()` from the migration.

```suggestion
  status!: PaymentBankAccountStatus
```

Reply:
Nice find, thanks!